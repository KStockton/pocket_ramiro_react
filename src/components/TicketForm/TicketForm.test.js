import React from 'react';
import { shallow } from 'enzyme';
import { TicketForm } from './';

describe('TicketForm', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<TicketForm/>);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});