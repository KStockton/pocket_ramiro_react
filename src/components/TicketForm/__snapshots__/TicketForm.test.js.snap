// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TicketForm should match the snapshot 1`] = `
<div
  className="form-container"
  id="form-ticket-container"
>
  <Formik
    enableReinitialize={false}
    initialValues={
      Object {
        "notes": "",
        "priority": "low",
      }
    }
    isInitialValid={false}
    onSubmit={[Function]}
    render={[Function]}
    validateOnBlur={true}
    validateOnChange={true}
    validationSchema={
      ObjectSchema {
        "_blacklist": RefSet {
          "list": Set {},
          "refs": Map {},
        },
        "_conditions": Array [],
        "_defaultDefault": [Function],
        "_deps": Array [],
        "_excludedEdges": Array [],
        "_exclusive": Object {},
        "_mutate": undefined,
        "_nodes": Array [
          "notes",
        ],
        "_options": Object {
          "abortEarly": true,
          "recursive": true,
        },
        "_type": "object",
        "_typeError": [Function],
        "_whitelist": RefSet {
          "list": Set {},
          "refs": Map {},
        },
        "fields": Object {
          "notes": StringSchema {
            "_blacklist": RefSet {
              "list": Set {},
              "refs": Map {},
            },
            "_conditions": Array [],
            "_deps": Array [],
            "_exclusive": Object {
              "min": true,
              "required": true,
            },
            "_mutate": undefined,
            "_options": Object {
              "abortEarly": true,
              "recursive": true,
            },
            "_type": "string",
            "_typeError": [Function],
            "_whitelist": RefSet {
              "list": Set {},
              "refs": Map {},
            },
            "tests": Array [
              [Function],
              [Function],
            ],
            "transforms": Array [
              [Function],
            ],
          },
        },
        "tests": Array [],
        "transforms": Array [
          [Function],
        ],
      }
    }
  />
</div>
`;
